src/components/InviteFriendsIcon.js

```diff
@@ -12,7 +12,8 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.text.secondary,
   },
   topLevel: {
-    marginRight: '11px',
+    paddingRight: theme.spacing(1),
```

Comment:
the default padding on the invite friends icon button is 12px, which was throwing everything off